satpy/etc/composites/mwr.yaml

```diff
@@ -0,0 +1,46 @@
+sensor_name: mwr
+
+composites:
+  mw183_humidity:
+    standard_name: mw183_humidity
+    compositor: !!python/name:satpy.composites.GenericCompositor
+    prerequisites:
+    - name: '11'
+    - name: '13'
+    - name: '15'
+
+  mw183_humidity_surface:
+    compositor: !!python/name:satpy.composites.RGBCompositor
+    prerequisites:
+    - name: '9'
+    - name: '10'
+    - name: '12'
+    standard_name: mw183_humidity_surface
+
+  mw325_humidity_surface:
+    compositor: !!python/name:satpy.composites.RGBCompositor
+    prerequisites:
+    - name: '9'
+    - name: '10'
+    - name: '19'
+    standard_name: mw325_humidity_surface
+
+  mw325_humidity:
+    compositor: !!python/name:satpy.composites.RGBCompositor
+    prerequisites:
+    - name: '16'
+    - name: '18'
+    - name: '19'
+    standard_name: mw325_humidity
+
+  ch1_tbs_colors:
+    compositor: !!python/name:satpy.composites.SingleBandCompositor
+    prerequisites:
+    - name: '1'
+    standard_name: tbs_colors
```

Comment:
I don't find a matching enhancement for this. Same in the `ch_tbs_colors` below.

Reply:
I will add, have them off line...

Reply:
Added now